src/docs/user-guide/running-jobs.md

```diff
@@ -157,7 +157,54 @@ account.
+
+***CPUs:*** How many CPUs the program your are calling the in the sbatch script needs, unless it can utilize multiple CPUs at once you should request a single CPU. Check your code's documentation or try running in an interactive session with sdev and run htop if you are unsure.
+
+***GPUs:*** If your code is GPU enabled, how many GPUs does your code need? Use the diagnostic tool [nvtop][url_nvtop] to see if your code is capable of running on multiple GPUs and how much GPU memory it's using in real-time.
+
+***memory (RAM):*** How much memory your job will consume. Some things to consider, will it load a large file or matrix into memory? Does it consume a lot of memory on your laptop? Often the default memory is sufficient for many jobs.
+
+***time:*** How long will it take for you code to run to completion?
+
+***partition:*** What set of compute nodes on Sherlock will you run on, normal, gpu, owners, bigmem? Use the [sh_part][url_sh_part] command to see what partitions you are allowed to run on. The default partition on Sherlock is the normal partition.
+
+Next, you tell the scheduler what your job should should do: load modules and run your code. Note that any logic you can code into a bash script with the [bash scripting language][url_bash] can also be coded into an sbatch script.
+
+This example job, will run the Python script mycode.py for 10 minutes on the normal partition using 1 CPU and 8 GB of memory. To aid in debugging we are naming this job "test_job" and appending the Job ID (%j) to the two output files that Slurm creates when a job is run. The output files are written to the directory in which you launched your job in, you can also specify a different path. One file will contain any errors and the other will contain non-error output.
+
+Because it's a Python 3 script that uses some Numpy code, we need to load the python/3.6.1 and the py-numpy/1.19.2_py36 modules. The Python script is then called just as you would on the command line at the end of the sbatch script:
+
+sbatch script:
+
+```#!/usr/bin/bash
+#!/usr/bin/bash
+#SBATCH --job-name=test_job
+#SBATCH --output=test_job.%j.out
+#SBATCH --error=test_job.%j.err
+#SBATCH --time=10:00
+#SBATCH -p normal
+#SBATCH -c 1
+#SBATCH --mem=8GB
+module load python/3.6.1
+module load py-numpy/1.19.2_py36
+python3 mycode.py
+```
+Create and edit the sbatch script with a [text editor][url_texteditors] like vim/nano or the [OnDemand file manager][url_filemanager]. Then save the file, in this example we call it "test.sbatch".
+
+Submit to the scheduler with the [sbatch][url_sbatch] command:
+
+```$sbatch test.sbatch
+```
+Monitor your job and job ID in the queue with the [squeue][url_squeue] command:
+
+```
+$squeue -u $USER
+```
+
+<script id="asciicast-n1bGQ06rHg9XmZfnFDafuwgSI" src="https://asciinema.org/a/n1bGQ06rHg9XmZfnFDafuwgSI.js" async></script>
```

Comment:
That's a pretty good idea, but maybe the text is a little small?
![image](https://user-images.githubusercontent.com/186807/146475788-bbbe678c-e32f-4a82-9fac-d0e7887d2460.png)